setup.py

```diff
@@ -1,14 +1,10 @@
-# --------------------------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# --------------------------------------------------------------------------------------------
-
 import ast
 import os
 import re
 import datetime
 
 from setuptools import setup, find_packages
+from codecs import open
```

Comment:
Why is this not needed?

Reply:
During the build process for RPM, rpm detects that it is a python package and uses it's built in python 2.7 to clean the build. This fails when it hits the long description of setup.py with a unicode decode error. The call to open requires a encoding utf-8 flag but is not supported on python 2. So instead we import built in codecs which handles the encoding for us.

Reply:
Interesting! Thanks.